src/components/toolbox/tooltip/tooltip.js

```diff
@@ -39,9 +39,9 @@ class Tooltip extends React.Component {
 
   handleMouseLeave() {
     this.setState({
-      timeoutObj: !this.state.clicked && this.state.showTooltip
-        && setTimeout(this.handleClose, 1000),
+      timeoutObj: !this.state.clicked && this.state.showTooltip,
```

Comment:
The solution work properly, however now that you remove the setTimeout then you need to remove the clearTimeout too, as we don't need it any more in the file